### ISSUE TITLE
Error after login.

### ISSUE THREAD

**Issue by airbat** [Teifun2/nextcloud-cookbook-flutter#175]
The app does'nt work since some cookbook updates.
The code must be corrected to remove some unused values.

I have make the changes and all is fine now and i can commit them now if you want.

**Comment by khaytsus** [comment]
Not sure if this is the error I'm seeing since there was no info, but I am getting the following...

Categories could not be loaded.  Make sure the Cookbook plugin is installed on this Nextcloud Instance!
Category load failed: Exception: FormatException: Unexpected character (at character 1) <!DOCTYPE HTML>

**Comment by airbat** [comment]
Yes this is the issue.
This is due to the links hardcoded in the app about API. The dev include the API v1 but it is not used by the cookbook nextcloud pluggin.

I have made all the changes and compiled the new app ans all is working again.

![Screenshot_2023-02-24-07-05-08-67_2d8abbdf9422cffd2b8dfb5abac3c4ae](https://user-images.githubusercontent.com/9461316/221104587-65a2713f-c224-454f-82e0-d90ad75a3360.jpg)

**Comment by khaytsus** [comment]
Wouldn't you fork, make changes in your fork, then make a pull request into this one?

**Comment by Teifun2** [comment]
@airbat If you have some code to share please create a PR i will gladly look at what you have changed.

**Comment by Leptopoda** [comment]
Hi @airbat can you still reproduce this issue with the app version 0.7.9 and the cookbook version 0.10.2 on your nextxloud?

There have been a lot of changes regarding the api recently.

If you are still facing the issue you could try the current develop branch?

Lastly please publish a log or the patch you've made. If you need assistance in how to achieve this just ask and I'll help.

Otherwise I'll close this issue with the next release.

**Comment by khaytsus** [comment]
My Nextcloud instance has Cookbook version 0.10.2 on it, the latest version, and your app still gives the same errors as it has for a while.

Categories could not be loaded. Make sure the Cookbook plugin is installed on this Nextcloud Instance!
Category load failed: Exception: FormatException: Unexpected character (at character 1) < ! DOCTYPE HTML >

EDIT:  Very strange.  I had used this app for a long time, no problems, then this error came up some months back.  Haven't touched the app since.  Today I had to clear app settings and create a new app password and login and it works.  I hadn't removed any app passwords, so not sure what would possibly have caused this?
